module-1/README.md

```diff
@@ -69,81 +80,245 @@ In the terminal, change directory to the newly cloned repository directory:
+#### Initialise CDK App folder
+
+Within the `module-1` folder create a new folder to contain your AWS CDK application
 
+```sh
+cd module-1
+mkdir cdk && cd cdk/
 ```
-aws s3 website s3://REPLACE_ME_BUCKET_NAME --index-document index.html
+
+In the `cdk` folder, lets now initialize a CDK app, where LANGUAGE is one of the supported programming languages: csharp (C#), java (Java), python (Python), or typescript (TypeScript) and TEMPLATE is an optional template that creates an app with different resources than the default app that cdk init creates for the language.
+
+_cdk init app --language LANGUAGE_
+
+For the purposes of this workshop we will use TypeScript as our language:
+
+```sh
+cdk init --language typescript
 ```
 
-#### Update the S3 Bucket Policy
+This command has now initialised a new CDK app in your `cdk` folder.  Part of the initialisation process also establishes the given directory as a new git repository.
 
-All buckets created in Amazon S3 are fully private by default.  In order to be used as a public website, we need to create an S3 [Bucket Policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html) that indicates objects stored within this new bucket may be publicly accessed by anyone. Bucket policies are represented as JSON documents that define the S3 *Actions* (S3 API calls) that are allowed (or not not allowed) to be performed by different *Principals* (in our case the public, or anyone).
+Notice the standard structure of an AWS CDK app, that consists of a `bin` folder and a `lib` folder.
 
-The JSON document for the necessary bucket policy is located at: `~/environment/aws-modern-application-workshop/module-1/aws-cli/website-bucket-policy.json`.  This file contains a string that needs to be replaced with the bucket name you've chosen (indicated with `REPLACE_ME_BUCKET_NAME`).  
+* The `bin` folder is where we will define the entry point for the CDK app.
+* The `lib` folder is where we will define all our workshop infrastructure components.
 
-To **open a file** in Cloud9, use the File Explorer on the left panel and double click `website-bucket-policy.json`:
+## Creating the Mythical Mysfits Static Website
 
-![bucket-policy-image.png](/images/module-1/bucket-policy-image.png)
+Now, let's define the infrastructure needed to host our static website.  
 
-This will open `bucket-policy.json` in the File Editor panel.  Replace the string shown with your chosen bucket name used in the previous commands:
+Create a new file called `static-website-stack.ts` in the `lib` folder, and define the skeleton class structure by writing/copying the following code:
 
-![replace-bucket-name.png](/images/module-1/replace-bucket-name.png)
+```typescript
+import cdk = require('@aws-cdk/core');
+
+export class StaticWebsiteStack extends cdk.Stack {
```

Comment:
The web app becomes an Angular based SPA and whilst its a static HTML page in module 1, the stack can be reused across all remaining modules.  Consider renaming to web-application-stack.ts as per below?

https://github.com/aws-samples/aws-modern-application-workshop/blob/dotnet-cdk/module-1/cdk/lib/web-application-stack.ts


Should we consider standardising on the naming of the stacks across the branches.

Reply:
Got it, will change this.